test/fixtures/issueList.json

```diff
@@ -15,6 +15,9 @@
                   }
                 ],
                 "totalCount": 1
+              },
+              "author": {
+                "login": "foo"
```

Comment:
Note: we are not requesting this field anymore, therefore we don't need this in stubbed results

Reply:
removed